eppoclient/rules_test.go

```diff
@@ -83,9 +97,9 @@ func Test_findMatchingRule_NumericValueAndRegex(t *testing.T) {
 }
 
 type MatchesAnyRuleTest []struct {
-	a    dictionary
-	b    []rule
-	expectedRule rule
+	a             dictionary
+	b             []rule
+	expectedRule  rule
```

Comment:
The mix of tabs here is giving me flashbacks to [Silicon Valley](https://www.youtube.com/watch?v=SsoOG6ZeyUI).